### PR TITLE
Rename and voidify timestamp getters

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/basic/time-util.c [hunk 1]
```diff
@@ -120,84 +120,62 @@ usec_t map_clock_usec(usec_t from, clockid_t from_clock, clockid_t to_clock) {
         return map_clock_usec_internal(from, now(from_clock), now(to_clock));
 }
 
-dual_timestamp* dual_timestamp_from_realtime(dual_timestamp *ts, usec_t u) {
+void dual_timestamp_from_realtime(dual_timestamp *ts, usec_t u) {
```
Comment (poettering) [inline src/basic/time-util.c]: why?

File: src/basic/time-util.c [hunk 2]
```diff
@@ -64,23 +64,19 @@ nsec_t now_nsec(clockid_t clock_id) {
...
 }
 
-dual_timestamp* dual_timestamp_get(dual_timestamp *ts) {
+void dual_timestamp_now(dual_timestamp *ts) {
         assert(ts);
 
         ts->realtime = now(CLOCK_REALTIME);
         ts->monotonic = now(CLOCK_MONOTONIC);
-
-        return ts;
```
Comment (yuwata) [inline src/basic/time-util.c:73]: Why this is dropped? Even if this is not used, I think it is useful in general.

**Review by yuwata — APPROVED** [review]
LGTM, except for the voidification.

File: src/basic/time-util.c (hunk 1 above)
Comment (keszybz) [reply src/basic/time-util.c]: I meant to put this in the commit message… None of the callers make use of this, so it seems not useful.

Comment (poettering) [reply src/basic/time-util.c]: but it's usually what we do in similar cases. I don't really see a reason to drop this tbh

**Comment by poettering** [comment]
so, lgtm, but the voidification I don't like.

**Comment by keszybz** [comment]
Voidification is dropped. (Part of the first commit and the second commit entirely.)

**Pull request merged**